app/assets/stylesheets/base/globals.scss

```diff
@@ -91,6 +91,10 @@ div.invalid-instance {
   flex: 1;
 }
 
+.flex-150 {
```

Comment:
I think we should create style for a table-like component. However, I prefered to keep existing usages.